Templates/JenkinsPipeline/Jenkinsfile

```diff
@@ -131,6 +131,21 @@ def BuildOutputDir            = ""         // DBB Build Output Directory (Discov
 def BuildFile                 = ""         // Will contain the contents of the DBB buildFile.txt.
 def BuildHasFiles             = true       // DBB buildFile.txt indicates changed files.
 
+// TAZ Unit Test results to JUnit
```

Comment:
The description should rather go into the README where we talk about install + setup.

Please make sure to document this new capability in the table of all pipeline implementations.